api4/license.go

```diff
@@ -210,3 +213,36 @@ func requestTrialLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func requestRenewalLink(c *Context, w http.ResponseWriter, r *http.Request) {
+	auditRec := c.MakeAuditRecord("requestRenewalLink", audit.Fail)
+	defer c.LogAuditRec(auditRec)
+	c.LogAudit("attempt")
+
+	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_SYSCONSOLE_READ_ABOUT) {
```

Comment:
Are the notifications for license renewal limited on this `PERMISSION_SYSCONSOLE_READ_ABOUT ` permission? The banner notifications will have the button to "Renew Now" and would need to generate a link

Reply:
I think it should. TBH I don't know how the notifications work, should be addressed in another PR, but it should only be shown to the admin so I think that permission should do the trick

Reply:
Looks odd to me that the user that has read rights (and possibly not write permissions) to the license section can generate the link. With the new System Roles, non-admins could have rights to `PERMISSION_SYSCONSOLE_READ_ABOUT`.

 I'm 1/5 that we should probably check `model.PERMISSION_SYSCONSOLE_WRITE_ABOUT` here

Reply:
I'll check that